client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -488,5 +453,7 @@ void main(void)
 	col = mix(skyBgColor, col, clarity);
 	col = vec4(col.rgb, base.a);
 
-	gl_FragColor = col;
+	gl_FragData[0] = col;
+	gl_FragData[1] = vec4((vNormal + 1.) / 2., DRAW_TYPE / 256.);
+	gl_FragData[2] = vec4((fogDistance-length(eyeVec))/1000, 0,0,0);
```

Comment:
the additional data is for the sake of the shadows shader, right?

Reply:
This is actually only used for debugging. Removed now